content/02.intro.md

```diff
@@ -1,10 +1,10 @@
 ## Introduction
 
-Deep learning (DL), a subfield of machine learning (ML) that focuses on deep artificial neural networks, is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
-However, DL itself remains an active area of research, and the complexity of this field poses a large barrier of entry to those who would like to correctly utilize state-of-the-art DL technology in biological research applications.
-While much has been written about the impressive results of DL in biology, there is a comparative dearth of literature articulating best practices for its use, with most instructional literature focusing on ML, rather than DL [@doi:10.1186/s13040-017-0155-3].
+Deep learning (DL), a subfield of machine learning (ML) implementing hierarchical artificial neural networks, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
```

Comment:
>implementing hierarchical artificial neural networks,

I am confused by the term "hierarchical artificial neural networks." I think this is way to specific. I think we should use sth like "artificial neural networks with many layers" or "deep artificial neural networks" is better. (hierarchical artificial neural networks refers to a special type of architecture, like these tree-structured neural nets that deep mind used for AlphaGo if I remember correctly)